src/style.css

```diff
@@ -1 +1,73 @@
-/* add styles here */
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  font-style: normal;
+  font-weight: normal;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-style: bold;
+  font-weight: bold;
```

Comment:
The 'font-weight' property doesn't need to be specified here. The browser will automatically assign a 'font-weight' of 'bold' to this font because you're loading a bold version of the font.